CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # developing
+- Rename function. [4608](https://github.com/beego/beego/pull/4608)
```

Comment:
Please move this to fix sonar section :)
![image](https://user-images.githubusercontent.com/9923838/118442497-3f5e2080-b71d-11eb-8e00-4075c4115aee.png)